includes/classes/order.php

```diff
@@ -325,6 +325,8 @@ function cart() {
     // set default tax calculation for not-logged-in visitors
       $taxCountryId = $taxZoneId = 0;
 
+    $billto = (!empty($_SESSION['billto']) ? (int)$_SESSION['billto'] : 0);
+    $sendto = (!empty($_SESSION['sendto']) ? (int)$_SESSION['sendto'] : 0);
```

Comment:
Should these then be moved to the top of the function, and used in the various queries that are run between there and here?